pyrefly/lib/alt/overload.rs

```diff
@@ -216,6 +216,31 @@ impl<'a, Ans: LookupAnswer> AnswersSolver<'a, Ans> {
         let args = call.vec_call_arg(args, self, errors);
         let keywords = call.vec_call_keyword(keywords, self, errors);
 
+        // Build a string showing the argument types for error messages
+        let mut arg_type_strs = Vec::new();
+        for arg in &args {
+            let (ty, prefix) = match arg {
+                CallArg::Arg(value) => (value.infer(self, errors), ""),
+                CallArg::Star(value, _) => (value.infer(self, errors), "*"),
+            };
+            let ty_display = self.solver().for_display(ty);
```

Comment:
nit: we have a convenient AnswersSolver wrapper for for_display, so you can just do `self.for_display(ty)`.